content/docs/lists-and-keys.md

```diff
@@ -202,13 +202,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**ทดลองบน CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+กฏของนิ้วโป้งที่ดีนั้น elements ใดๆที่ถูกเรียกภายใน `map()` นั้นจำเป็นต้องมี key
```

Comment:
here is my suggestion
กฏของนิ้วโป้งที่ดีนั้น > หลักการง่ายๆ, หลักการทั่วๆไป, กฎโดยทั่วไป or something similar